docs/src/content/docs/zh-cn/guides/components.mdx

```diff
@@ -295,3 +269,117 @@ import importedCode from '/src/env.d.ts?raw';
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### 文件树
+
+使用`<FileTree>`组件来显示带有文件图标和可折叠子目录的目录结构。
+
+使用 `<FileTree>` 组件内置的 [Markdown 无序列表语法](https://www.markdownguide.org/basic-syntax/#unordered-lists)指定文件和目录的组织结构。使用嵌套的列表项创建子目录；若希望某个目录不显示具体内容，只需在列表项末尾添加斜杠`/`即可。
+
+以下语法可用于自定义文件树的外观：
+
+- 通过加粗文件名或目录名来突出显示，例如 `**README.md**`
+- 通过在名称后添加更多文本来为文件或目录添加注释
+- 使用 `...` 或 `…` 作为名称来添加占位符文件和目录。
+
+```mdx
+# src/content/docs/example.mdx
+
+import { FileTree } from '@astrojs/starlight/components';
+
+<FileTree>
+
+- astro.config.mjs an **important** file
```

Comment:
The comment `an **important** file` can be safely translated. Would it make sense to do so?

I'll let you decide as you definitely know the best but if yes, the example below should also be updated.

Reply:
Yes, that makes sense, thank you for the advice, I will deal with these issues.